examples/videoplayer.jl

```diff
@@ -212,4 +212,59 @@
 
         RecordEvents(scene, @replace_with_a_path(mp4))
     end
+
+    @cell "VFR test video" begin
+        using Dates, Observables, Printf
+
+        tosecond(x::T) where {T} = x/convert(T, Second(1))
+
+        dar = 2
+        w = 200
+        h = dar*w
+        Δt = Millisecond(1):oneunit(Millisecond):Millisecond(w)
+        ts = cumsum(Δt)
+        # totaltime = ts[end]
+        frame = Observable(1)
+        point = lift(frame) do fn
+            [(fn, 2fn)]
+        end
+        msg = lift(frame, point) do fn, p
+            ti = @sprintf "%03i" fn
+            t = @sprintf "%-012s" Time(0) + ts[fn]
+            x, y = p[]
+            """
+        Frame #: $ti
+        Time: $t
+        x: $x
+        y: $y"""
+        end
+        limits = FRect(1, 2, w - 1, 2w - 2)
+        scene = Scene(resolution = (w, h), limits = limits,  scale_plot = false, show_axis = false)#, padding=(0,0), backgroundcolor=:red)
+        poly!(scene, limits, color = :gray)
+        text!(scene, msg, position = (2, w), align = (:left,  :bottom))#, color = :white)
+        scatter!(scene, point, markersize = 10, marker = :+)#, color = :white)
+        for i in 1:w
+            frame[] = i
+            AbstractPlotting.update_limits!(scene)
+            AbstractPlotting.update!(scene)
+            fname = @sprintf "%03i.png" i
+            save(fname, scene)
+        end
+        run(`mogrify -resize $(h)x$h\! "*".png`)
```

Comment:
@yakir12 any reason for this?  It's the only thing blocking this from being runnable with stock Makie.

Reply:
this is a hack to morph the images to a different S/PAR so that the video will have the intended PAR and the resulting display will look right.

Reply:
Cool, I'll remove it then.